SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -0,0 +1,234 @@
+                                SKIP_GO_VERSION_CHECK=1
+
+%global agent_make_vars         LIBC=gnu
+
+%global debug_package %{nil}
+
+Name:         kata-containers-cc
+Version:      0.1.0
+Release:      10%{?dist}
+Summary:      Kata Confidential Containers
+License:      ASL 2.0
+Vendor:       Microsoft Corporation
+URL:          https://github.com/microsoft/kata-containers
+Source0:      https://github.com/microsoft/kata-containers/archive/refs/tags/cc-%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Source1:      https://github.com/microsoft/kata-containers/archive/refs/tags/%{name}-%{version}.tar.gz
+
+Source2:      mariner-coco-build-uvm-image.sh
+Source3:      mariner-coco-build-uvm-image.service
+Source4:      containerd-for-cc-override.conf
+Source5:      runtime.yaml
+Source6:      pause-image.sh
+Source7:      pause-image.service
+
+ExclusiveArch: x86_64
+
+BuildRequires:  golang
+BuildRequires:  make
+BuildRequires:  systemd
+BuildRequires:  gcc
+BuildRequires:  protobuf-compiler
+BuildRequires:  mariner-release
+BuildRequires:  dracut
+BuildRequires:  busybox
+BuildRequires:  cargo
+BuildRequires:  rust
+BuildRequires:  git
+BuildRequires:  sudo
+BuildRequires:  curl
+BuildRequires:  openssl-devel
+BuildRequires:  iptables-devel
+BuildRequires:  wget
+BuildRequires:  perl-FindBin
+BuildRequires:  perl-lib
+BuildRequires:  libseccomp-devel
+BuildRequires:  kernel-uvm-devel
+BuildRequires:  parted
+BuildRequires:  opa >= 0.50.2
+
+Requires:  systemd
+Requires:  iptables
+Requires:  parted
+Requires:  qemu-img
+Requires:  opa
+Requires:  kernel-uvm
+
+%description
+Kata Confidential Containers.
+
+%prep
+%autosetup -p1 -n %{name}-%{version}
+
+%build
+export PATH=$PATH:"$(pwd)/go/bin"
+export GOPATH="$(pwd)/go"
+
+# Runtime
+pushd %{_builddir}/%{name}-%{version}/src/runtime
+%make_build %{runtime_make_vars}
+popd
+
+# Tardev snapshotter
+pushd %{_builddir}/%{name}-%{version}/src/tardev-snapshotter
+make
+chmod +x target/release/tardev-snapshotter
+popd
+
+pushd /usr/src/linux-headers*
+header_dir=$(basename $PWD)
+KERNEL_VER=${header_dir#"linux-headers-"}
+KERNEL_MODULE_VER=${KERNEL_VER%%-*}
+popd
+
+# Kernel modules
+pushd %{_builddir}/%{name}-%{version}/src/tarfs
+make KDIR=/usr/src/linux-headers-$KERNEL_VER
+make KDIR=/usr/src/linux-headers-$KERNEL_VER install
+popd
+%define KERNEL_MODULES_DIR %{_builddir}/%{name}-%{version}/src/tarfs/_install/lib/modules/$KERNEL_MODULE_VER
+
+# Agent
+pushd %{_builddir}/%{name}-%{version}/src/agent
+%make_build %{agent_make_vars}
+popd
+
+# UVM rootfs
+pushd %{_builddir}/%{name}-%{version}/tools/osbuilder
+gcc -O2 ./image-builder/nsdax.gpl.c -o ./image-builder/nsdax
+
+sudo -E PATH=$PATH \
+ GOPATH="$(sudo readlink -f %{_builddir}/%{name}-%{version}/src/runtime/go)" \
+ AGENT_SOURCE_BIN="$(sudo readlink -f %{_builddir}/%{name}-%{version}/src/agent/target/x86_64-unknown-linux-gnu/debug/kata-agent)" \
+ make DISTRO=mariner KERNEL_MODULES_DIR=%{KERNEL_MODULES_DIR} rootfs
```

Comment:
So in rebased cc-msft, I *think* the distro is called cbl-mariner https://github.com/microsoft/kata-containers/commit/36198274dcb4332f1acd445d2a80854232b1d236. If merged, just heads up this might need to be changed in the near future, or we'll run into 
<img width="739" alt="image" src="https://user-images.githubusercontent.com/30801614/230513712-e9f2f84e-f6be-40ab-82e1-b0ee2426f024.png">

Reply:
That's a good callout, it will have to be changed in the for the next kata-containers release we consume. For now the source release is pointing at the non-rebased code